specification/_types/Retriever.ts

```diff
@@ -133,9 +133,25 @@ export class KnnRetriever extends RetrieverBase {
   rescore_vector?: RescoreVector
 }
 
+/**
+ * Wraps a retriever with an optional weight for RRF scoring.
+ */
+export class RRFRetrieverComponent {
+  /** The nested retriever configuration. */
+  retriever: RetrieverContainer
+  /** Weight multiplier for this retriever's contribution to the RRF score. Higher values increase influence. Defaults to 1.0 if not specified. Must be non-negative. */
```

Comment:
/nit: we should also add the @server_default tag as a machine readable alternative to mentioning the default value in the doc string.